go/vt/schemamanager/tablet_executor.go

```diff
@@ -416,7 +416,7 @@ func (exec *TabletExecutor) executeOneTablet(
 	if viaQueryService {
 		result, err = exec.wr.TabletManagerClient().ExecuteQuery(ctx, tablet, []byte(sql), 10)
 	} else {
-		result, err = exec.wr.TabletManagerClient().ExecuteFetchAsDba(ctx, tablet, false, []byte(sql), 10, false, true)
+		result, err = exec.wr.TabletManagerClient().ExecuteFetchAsDba(ctx, tablet, false, []byte(sql), 10, false, false, true)
```

Comment:
Should we start commenting each bool value with the formal argument name?
To avoid coding errors where values are incorrectly transposed.